frontend/packages/pipelines-plugin/src/components/repository/sections/RepositoryOverview.tsx

```diff
@@ -88,7 +98,27 @@ const RepositoryOverview = () => {
           {t('pipelines-plugin~Commit these changes and push them to your Git repository.')}
         </Text>
       </FormGroup>
-      <FormGroup fieldId="step-4">
+      {!(
+        values.gitProvider === GitProvider.GITHUB &&
+        values.method === GitProvider.GITHUB &&
+        values.githubAppAvailable === true
+      ) && (
+        <FormGroup fieldId="step-4">
+          <Title headingLevel="h4" size={TitleSizes.md}>
+            {t('pipelines-plugin~Step 4')}
+          </Title>
+          <Text>
+            <Trans t={t} ns="pipelines-plugin">
+              Webhook URL to configure the webhook in your{' '}
+              <a href={values.gitUrl}>Git repository</a> :
+            </Trans>
```

Comment:
We link Git Repository once, and all "Git Repo" in the steps are not linked anymore, so I would not do that here.

If you prefer a link here then please use a <ExternalLink> and remove the space between Git repository and `:`

```suggestion
            <Trans t={t} ns="pipelines-plugin">
              Webhook URL to configure the webhook in your Git repository:
            </Trans>
```